e3sm_to_cmip/mpas.py

```diff
@@ -338,6 +338,8 @@ def setup_cmor(varname, tables, user_input_path, component='ocean'):
         table = 'CMIP6_Omon.json'
     elif component == 'seaice':
         table = 'CMIP6_SImon.json'
+    elif component == "ocean_fixed":
+        table = 'CMIP6_Ofx.json'
```

Comment:
@sterlingbaldwin, I think I would opt of another solution where we add an optional argument `table` or similar and leave the component unchanged.  Treating `ocean_fixed` as a new component just in this once instance seems like a bad hack to me.

Reply:
Yes I agree, especially since the TABLE variable is available from all the handlers. Since the component variable is just used to switch between the CMIP table, I think the correct table should be passed in from the handler.

Reply:
I'll make this change, just so we don't step on each other's toes.

Reply:
if you switch from mpas.setup_cmor to e3sm_to_cmip.util.setup_cmor you can remove mpas.setup_cmor

Reply:
Okay, done.